tools/mo/requirements.txt

```diff
@@ -5,7 +5,8 @@ mxnet~=1.7.0.post2; sys_platform != 'win32'
 networkx~=2.5; python_version <= "3.6"
 networkx<2.8.1; python_version > "3.6"
 protobuf>=3.18.1,<4.0.0
-onnx>=1.8.1,<1.12
+onnx==1.11.0; python_version == "3.6*"
```

Comment:
Why is "3.6" kept? Only for compatibility with master CIs?

Reply:
It must have came from merging https://github.com/openvinotoolkit/openvino/pull/12847 into this branch. It's not needed here, I deleted it.